src/components/big-button/big-button.css

```
@@ -1,4 +1,17 @@
 .BigButton {
-    border: 1px solid white;
+    height: 105px;
+    max-width: 312px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.enabled {
     background-color: var(--color-bg-secondary);
+    color: white;
+}
+
+.disabled {
```

prefix with `.BigButton` because styles will be added to global scope.

```suggestion
.BigButton.disabled {
```